pkg/cloudprovider/aws/fleet/packing/packing.go

```diff
@@ -16,17 +16,21 @@ package packing
 
 import (
 	"context"
+	"errors"
+	"fmt"
+	"sort"
 
 	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
 	"go.uber.org/zap"
 	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 )
 
-type PodPacker struct {
+type podPacker struct {
 	ec2 ec2iface.EC2API
```

Comment:
Is `ec2` being used right now?

Reply:
Not right now, but will be used shortly.

Reply:
I'd add a TODO here just to make sure that's clear